shapeFunctions/script.js

```diff
@@ -0,0 +1,33 @@
+// Area of Circle
+function getAreaOfCircle(radius) {
+    return Math.PI * radius**2
+}
+const x = getAreaOfCircle(5)
```

Comment:
I would condense this into one line, like you have on lines 29 and 33. 

console.log(getAreaOfCircle(5)